docs/HazardAnalysis/HazardAnalysis.tex

```diff
@@ -150,14 +150,11 @@ \subsection{Failure Mode and Effects Analysis Table}
+        Viewing AR objects accurately in the Realm screen & Location access is disabled & User is unable to accurately view object instances in their surroundings & Permission for location denied by mobile device, Location access disabled by the user, Bugs in software component synchronizing object and device location with each other & Having periodic updates of device location & Prompting user to grant location access or transitioning to an offline view of the Realm screen & \ref{ACR-1}, \ref{ACR-2} & H3-1 \\
         \hline
-        Navigating to AR object cluster & AR objects within selected cluster are deleted by the owner during navigation & User may arrive at destination with no AR objects present, Bugs in navigation software may cause app crashes & All AR objects within the targeted cluster are deleted by respective owners & Keeping track of AR object cluster count & Before starting navigation, check for existence of AR object cluster. Notify user about objects being deleted by owners. Provide option to start navigation back to original starting point & ACR-3, ROR-1 & H4-1 \\
-        & Location access is disabled & System is unable to present current user location, system is unable to display next instruction & Location access disabled by the user or device system & Having periodic updates of device location & Prompt user to grant location access to continue navigation & ACR-2 & H4-2 \\
+        Navigating to AR object cluster & AR objects within selected cluster are deleted by the owner during navigation & User may arrive at destination with no AR objects present, Bugs in navigation software may cause app crashes & All AR objects within the targeted cluster are deleted by respective owners & Keeping track of AR object cluster count & Before starting navigation, check for existence of AR object cluster. Notify user about objects being deleted by owners. Provide option to start navigation back to original starting point & \ref{ACR-3}, \ref{ROR-1} & H4-1 \\
+        & Location access is disabled & System is unable to present current user location, system is unable to display next instruction & Location access disabled by the user or device system & Having periodic updates of device location & Prompt user to grant location access to continue navigation & \ref{ACR-2} & H4-2 \\
+        & System navigates user to AR object cluster located in a restricted area/non-public area & User would be considered as unauthorized personnel & System fails to distinguish between restricted and non-restricted areas & Have a record of restricted/private areas & System denies navigation to the area by notifying the user about the area destination being restricted to the general public & \ref{PVR-1} & H4-3 \\
```

Comment:
effects of failure should be expanded upon, maybe talk about specific scenarios, (trespassing, physical danger if its a construction site or something, time wastage if the area is inaccessible)

Reply:
Expanded on the effects of failures